noir-projects/noir-contracts/contracts/test/txe_test_contract/src/test.nr

```diff
@@ -1,50 +1,50 @@
 use crate::TXETest;
 
-use dep::aztec::test::helpers::test_environment::TestEnvironment;
-
-use aztec::macros::functions::initializer;
+use aztec::{
+    macros::functions::initializer,
+    note::{note_getter::{MAX_NOTES_PER_PAGE, view_notes}, note_viewer_options::NoteViewerOptions},
+    oracle::notes::set_sender_for_tags,
+    protocol_types::{address::AztecAddress, storage::map::derive_storage_slot_in_map},
+    test::helpers::test_environment::TestEnvironment,
+};
 use value_note::{balance_utils, value_note::ValueNote};
 
-use dep::aztec::note::note_getter::{MAX_NOTES_PER_PAGE, view_notes};
-use dep::aztec::note::note_viewer_options::NoteViewerOptions;
-use dep::aztec::protocol_types::{address::AztecAddress, storage::map::derive_storage_slot_in_map};
-
 pub unconstrained fn setup(
-    initial_value: Field,
-) -> (&mut TestEnvironment, AztecAddress, AztecAddress, AztecAddress) {
+    initial_value: u64,
+) -> (&mut TestEnvironment, AztecAddress, AztecAddress) {
     // Setup env, generate keys
-    let mut env = TestEnvironment::_new();
+    let mut env = TestEnvironment::new();
     let owner = env.create_account(1);
-    let sender = env.create_account(2);
+
+    // We are not deploying any account contracts and so we need to manually set the sender for tags. This is
+    // usually done by account contract's entrypoint.
+    let sender = env.create_account(7097070);
+    set_sender_for_tags(sender);
+
     env.impersonate(owner);
 
     // Deploy contract and initialize
-    let initializer = TXETest::interface().initialize(initial_value as u64, owner);
+    let initializer = TXETest::interface().initialize(initial_value, owner);
     let counter_contract = env.deploy_self("TXETest").with_private_initializer(owner, initializer);
     let contract_address = counter_contract.to_address();
-    (&mut env, contract_address, owner, sender)
+    (&mut env, contract_address, owner)
 }
 
 #[test]
 unconstrained fn new_calling_flow_increment_self_and_other() {
-    let mut env = TestEnvironment::new();
-    let owner = env.create_account(1);
-    let sender = env.create_account(2);
-    let third = env.create_account(3);
+    let initial_value: u64 = 2;
```

Comment:
Same as before. `setup` util was not used here and below for seemingly no reason.